test/cases/mount-bpf-fs.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export MOUNT_BPF_FS_DEBUG=true
+
+echo "--> Should succeed if bpf type fs already exists"
+function mount() {
+  echo "none on /foo/bar type bpf (rw,nosuid,nodev,noexec,relatime,mode=700)"
+}
+export -f mount
```

Comment:
Fancy trick. Wasn't familiar with this one.

Reply:
Yeah I think this is the right approach to "mocking" in these test cases, but I'm not going to refactor them all in this PR 😂